### ISSUE TITLE
[BUG] Crash Exporting to Kitsu

### ISSUE THREAD

**Issue by cpiber** [Glagan/SyncDex#2]
**Describe the bug**
Crashes when exporting to Kitsu, particularly on [Toriko](https://mangadex.org/title/107/toriko) (on my plan-to-read if that makes a difference).

**To Reproduce**
1. Export to Kitsu

**Screenshots**
![Screenshot_2020-12-23_19-28-26](https://user-images.githubusercontent.com/59023762/103027984-2e86f580-4557-11eb-9052-8278d3b3bffa.png)
![Screenshot_2020-12-23_19-29-45](https://user-images.githubusercontent.com/59023762/103028011-3d6da800-4557-11eb-9db1-803d3486ca74.png)
![Screenshot_2020-12-23_19-31-11](https://user-images.githubusercontent.com/59023762/103028017-3fd00200-4557-11eb-9c99-4de29379c93c.png)
![Screenshot_2020-12-23_19-41-00](https://user-images.githubusercontent.com/59023762/103028028-42325c00-4557-11eb-8ba9-aebf29aa7f64.png)
![Screenshot_2020-12-23_19-41-27](https://user-images.githubusercontent.com/59023762/103028031-43fc1f80-4557-11eb-99a1-30e8459a8c62.png)

**Browser**
Firefox 84.0

**Options and logs**
```
{"logs":[{"d":1608744434310,"msg":"Installation version 0.1"},{"d":1608747614175,"msg":"TypeError: this.key is undefined\nStack: KitsuTitle/this.persist@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:4281:37\nKitsuExport/this.execute@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:4173:46\n"},{"d":1608748158345,"msg":"TypeError: this.key is undefined\nStack: KitsuTitle/this.persist@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:4281:37\nKitsuExport/this.execute@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:4173:46\nasync*ExportModule/this.run@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:3249:43\nasync*Module/this.bindInterface/<@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:2877:58\nModuleInterface/this.bindFormSubmit/<@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:5344:21\n"},{"d":1608748233353,"msg":"TypeError: this.key is undefined\nStack: KitsuTitle/this.persist@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:4281:37\nKitsuExport/this.execute@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:4173:46\nasync*ExportModule/this.run@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:3249:43\nasync*Module/this.bindInterface/<@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:2877:58\nModuleInterface/this.bindFormSubmit/<@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:5344:21\nEventListener.handleEvent*ModuleInterface/this.bindFormSubmit@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:5337:23\nModule/this.bindInterface@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:2877:32\nExportModule@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:3268:14\nKitsuExport@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:4123:9\nKitsu.exportModule@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:4197:43\nServiceCard/this.bind/<@moz-extension://33477aff-cb89-4241-a0c9-349e02f7edb0/options/SyncDex_options.js:5647:51\n"}],"options":{"hideHigher":false,"hideLower":false,"hideLast":false,"highlight":true,"groupTitlesInLists":true,"thumbnail":true,"originalThumbnail":false,"progressInThumbnail":true,"thumbnailMaxHeight":80,"separateLanguages":true,"favoriteLanguage":"gb","saveOpenedChapters":false,"chaptersSaved":100,"saveOnlyHigher":true,"saveOnlyNext":true,"confirmChapter":true,"updateOnlyInList":true,"linkToServices":true,"overviewMainOnly":true,"autoSync":true,"mdUpdateSyncDex":false,"biggerHistory":false,"chapterStatus":false,"notifications":true,"errorNotifications":true,"useMochi":true,"acceptLowScore":false,"updateMD":true,"checkOnStartup":false,"checkOnStartupMainOnly":true,"checkOnStartupCooldown":30,"services":["mal","mu","ku"],"mainService":"mal","noReloadStatus":true,"tokens":{"kitsuToken":"set","kitsuUser":"set"},"colors":{"highlights":["rgba(28, 135, 141, 0.8)","rgba(22, 65, 87, 0.8)","rgba(28, 103, 141, 0.8)"],"nextChapter":"rgba(199, 146, 2, 0.4)","higherChapter":"transparent","lowerChapter":"rgba(180, 102, 75, 0.4)","openedChapter":"rgba(28, 135, 141, 0.4)"},"version":0.1,"subVersion":1}}
```

**Comment by Glagan** [comment]
Should be fixed with ``0.1.2`` that's up for **Firefox**.

**Comment by cpiber** [comment]
Yes works now thank you
